packages/cli/src/commands/stream/commits.ts

```diff
@@ -1,22 +1,23 @@
-import { Command, type CommandFlags } from '../../command.js'
+import { Command, STREAM_ID_ARG, SYNC_ARG, SYNC_OPTIONS_MAP, type CommandFlags } from '../../command.js'
 
-export default class Commits extends Command<CommandFlags, { streamId: string }> {
+export default class Commits extends Command<CommandFlags, { streamId: string, sync: string }> {
   static description = 'list commits contained within a stream'
 
   static args = [
-    {
-      name: 'streamId',
-      description: 'ID of the stream',
-      required: true,
-    },
+    STREAM_ID_ARG,
+    SYNC_ARG,
   ]
 
   async run(): Promise<void> {
     this.spinner.start(`Loading stream ${this.args.streamId}...`)
     try {
-      const stream = await this.ceramic.loadStream(this.args.streamId)
+      const syncArg = SYNC_OPTIONS_MAP[this.args.sync]
+      const stream = await this.ceramic.loadStream(
+        this.args.streamId,
+        this.args.sync !== undefined ? { sync: syncArg} : undefined,
+      )
       const commits = stream.allCommitIds.map((v) => v.toString())
-      this.spinner.succeed('Stream commits loaded.')
+      this.spinner.succeed(`Stream commits loaded.${syncArg !== undefined ? `Syncing option chosen: ${this.args.sync}` : ``}`)
```

Comment:
I don't think it's useful to explicitly indicate this, it's just displaying an input rather than output information.